queries.sql

```diff
@@ -1 +1,8 @@
-
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Greate work on selecting the names from the animals table however the date should be between '2016-01-01' AND '2019-31-12', kindly make sure to change the last date accordingly

Reply:
Changes have been made as per instructions